docs/notebooks/pivoted_document_length_normalisation.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py:41: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.\n",
+      "  \"This module will be removed in 0.20.\", DeprecationWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "from sklearn.datasets import fetch_20newsgroups\n",
+    "from sklearn.cross_validation import train_test_split\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "\n",
+    "from gensim.corpora import Dictionary\n",
+    "from gensim.sklearn_api.tfidf import TfIdfTransformer\n",
+    "from gensim.matutils import corpus2csc\n",
+    "\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as py\n",
+    "import matplotlib.animation as animation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cats=['sci.space', 'comp.graphics']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "newsgroups = fetch_20newsgroups(subset='train',\n",
+    "                                categories=['sci.space', 'comp.graphics'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test = train_test_split(newsgroups.data, newsgroups.target, random_state=49)\n",
+    "\n",
+    "id2word = Dictionary([_.split() for _ in X_train])\n",
+    "\n",
+    "train_corpus = [id2word.doc2bow(i.split()) for i in X_train]\n",
+    "test_corpus = [id2word.doc2bow(i.split()) for i in X_test]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(882, 295)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print (len(X_train), len(X_test))"
```

Comment:
no need to add space between `print` and `(`, please check carefully PEP8 in your notebook.